### PR TITLE
Add resource centre to nav pages AB#10535

### PR DESCRIPTION
**Pull request by MikeLyttle**
# Fixes or Implements [AB#10535](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/10535)

-   [x] Enhancement
-   [ ] Bug
-   [ ] Documentation

## Description

Adds a button in the bottom-right corner of all pages that are included in the left navigation (Timeline, Dependents, Export Records, Health Insights). This button opens the resource centre menu, containing links to the FAQ and Release Notes and a button to display the COVID-19 Card.

Also removes the FAQ link from the left-hand navigation, refactors the filtering for COVID-19 immunizations to a getter, and adds a property to the HgIcon component that can force icons to be square.

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

-   [ ] Unit Tests Updated
-   [ ] Functional Tests Updated
-   [ ] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

YES

### Browsers Tested

-   [X] Chrome
-   [ ] Safari
-   [ ] Edge
-   [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant. Include any specialized deployment steps here.

### CODE REVIEW

**Review by tiagograf-AOT — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/components/resourceCentre.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                >COVID-19 Card</b-dropdown-item-button
+            >
+            <b-dropdown-item to="/faq">FAQ</b-dropdown-item>
+            <b-dropdown-item to="/release-notes">Release Notes</b-dropdown-item>
+        </b-dropdown>
+    </div>
+</template>
+
+<style lang="scss">
+// deliberately unscoped to affect button styling in dropdown
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/components/resourceCentre.vue:58]: nit: no need fore the comment

File: Apps/WebClient/src/ClientApp/src/views/dependents.vue [hunk 2]
```diff
@@ -84,7 +86,7 @@ export default class DependentsView extends Vue {
 }
 </script>
 <template>
-    <div class="m-3">
+    <div class="m-3 flex-grow-1 d-flex flex-column">
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/dependents.vue:89]: Q - Why is this necessary? the resource center can float without having to interact with the rest of the layoutt

**Review by FuriousLlama — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/components/resourceCentre.vue (hunk 1 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/src/components/resourceCentre.vue:58]: right, we have that elsewhere as well. IMO that is implied when scoped is not there

File: Apps/WebClient/src/ClientApp/src/views/dependents.vue (hunk 2 above)
Comment (MikeLyttle) [reply Apps/WebClient/src/ClientApp/src/views/dependents.vue:89]: The resource centre is sticky rather than abolutely positioned.  It's located at the bottom of the page content.  If the parent is not tall enough, the resource centre will not appear in the bottom right - it will appear in the middle-right where the parent content ends.

The .flex-grow allows the parent to grow to fill the remaining space that is provided by _its_ parent's .fill-height.  Applying .d-flex to it then lets us position the resource centre at the bottom of it using .mt-auto.
